sycamore/tests/unit/transforms/test_spread_properties.py

```diff
@@ -0,0 +1,37 @@
+from sycamore.data import Document
+from sycamore.transforms import SpreadProperties
+
+
+class TestSpreadProperties:
```

Comment:
We should also add a test which actually operates on the dataset to verify the underlying Ray calls.

Reply:
I looked at the other tests and they're very confusing with all the mocking calls.  Do we need to repeat that testing here?

Reply:
Do we need mocking. Can we just run the transform on one of the docs we have committed (or synthetic data)?

Reply:
I am testing the logic, but just not through Ray.  Maybe someone can show me a straightforward way to test more of the wrapper code.

Reply:
I'll add a unit test that uses execute().